.travis/sync.py

```diff
@@ -216,7 +248,9 @@ def main():
     # Download zip
     url = args.url
     userzip = True
-    if urllib.parse.urlparse(url).scheme != '':
+    if urllib.parse.urlparse(url).scheme == 'd':
```

Comment:
On windows, the scheme for a file path as url is 'd', but I did not find the documentation for that in the urllib doc. I did not check if it was the case on Linux, but this part of the code was working on Linux.